### DISCUSSION TITLE
chat breaks every time someone changes a team

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#14959] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

when someone changes their team in pvp, everyone who was  chatting gets their text box cleared
this makes chat somewhat unsuable and incredibly frustrating

### Reproduction steps

1. pvp mode
2. start typing something
3. someone changes team?

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next hotfix update. Closing discussion.
